.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: wontfix
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
```

Comment:
Perhaps "any activity _in the past 60 days_".

Reply:
I left this without a number to prevent it falling out of sync with the param above.

Reply:
I left this without a number to prevent it falling out of sync with the param above.